ophyd/utils/epics_pvs.py

```diff
@@ -414,7 +414,8 @@ def set_and_wait(signal, val, poll_time=0.01, timeout=10, rtol=None,
         logger.info("Waiting for %s to be set from %r to %r%s...",
                     signal.name, current_value, val, within_str)
         ttime.sleep(poll_time)
-        poll_time *= 2  # logarithmic back-off
+        if poll_time < 0.1:
+            poll_time *= 2  # logarithmic back-off
```

Comment:
Is a `poll_time` of 0 valid?
Perhaps something like this would be advisable:
`poll_time = max(poll_time * 2, min_poll_time)`